Date/Span.php

```diff
@@ -1087,7 +1087,7 @@ function isEmpty()
      * @static
      * @see      Date_Span::getDefaultInputFormat(), Date_Span::setDefaultFormat()
      */
-    function setDefaultInputFormat($format)
+    static function setDefaultInputFormat($format)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.